### PR TITLE
[Merged by Bors] - chore: avoid some unused variables

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
These will be caught by the linter in a future lean version.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/Finite/Basic.lean [hunk 1]
```diff
@@ -204,13 +204,10 @@ theorem sum_subgroup_pow_eq_zero [CommRing K] [NoZeroDivisors K]
...
     rw [sub_mul, mul_comm, ← h_multiset_map_sum, one_mul, sub_self]
   rw [mul_eq_zero] at hzero
-  rcases hzero with h | h
-  · contrapose! ha
-    ext
-    rw [← sub_eq_zero]
-    simp_rw [SubmonoidClass.coe_pow, Units.val_pow_eq_pow_val, OneMemClass.coe_one,
-      Units.val_one, h]
-  · exact h
+  refine hzero.resolve_left fun h => ha ?_
```
Comment (grunweg) [inline Mathlib/FieldTheory/Finite/Basic.lean:207]: Use "\maps" instead of "=>" while you're at it?

**Review by grunweg — APPROVED** [review]
LGTM. Just one nit, which is not important.

**Review by jcommelin — APPROVED** [review]
bors d+

**Comment by semorrison** [comment]
bors merge

File: Mathlib/FieldTheory/Finite/Basic.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/FieldTheory/Finite/Basic.lean:207]: (Not official or anything, but my preference is against "while you're at it" suggestions. They really do, in my opinion, add friction to getting stuff done.)

Comment (grunweg) [reply Mathlib/FieldTheory/Finite/Basic.lean:207]: I understand this point!
My point was this is a *new* proof written - so reviewing doesn't become any harder this way. *Authoring* it does; thanks for pointing this out.

Comment (grunweg) [reply Mathlib/FieldTheory/Finite/Basic.lean:207]: Let me add: I can make sure to, in the future,
- make such comments directly as suggestions (minimizing the overhead for authors and reviewers)
- limit these comments to new proofs, or PRs cleaning up miscellaneous things, where such a change is in scope.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**